### DISCUSSION TITLE
Error when using coreui with webpack and bootstrap admin template

### DISCUSSION THREAD

**Discussion by geraldhanks** [coreui/coreui#354] [Q&A]
I have installed webpack into my project and have included coreui in my app.js file using "import '@coreui/coreui'". Webpack builds my javascript bundle but I seem to be missing something since I am getting an error when using the bootstrap admin template and trying to hide the sidebar with onclick="coreui.Sidebar.getInstance(document.querySelector('#sidebar')).toggle()"

The sidebar fails to toggle and my browser debugger shows that the variable 'coreui' cannot be found.

Any ideas on what I could be missing?

**Comment by mrholek** [comment]
@geraldhanks  Can you try this `import * as coreui from '@coreui/coreui';`

**Reply by kantorge** [reply]
Thank you for the suggestion. I've been able to get it working with the following code.

`import * as coreui from '@coreui/coreui';
window.coreui = coreui;`

**Reply by jheathco** [reply]
This worked for me as well - thank you.  Wonder why this is necessary?

**Comment by foarsitter** [comment]
Don't use the inline onclick, just add some plain Javascript code in your app.js and add an event listener. 

It is another way of thinking but you almost never need the inline stuff.
